### PR TITLE
Removed all LOG lines

### LINKED ISSUE #8: Remove all log lines

### PR DESCRIPTION
**Pull request by Medhavi-16**

### CODE REVIEW

File: library/src/main/java/in/krharsh17/barview/Bar.java [hunk 1]
```diff
@@ -44,7 +44,8 @@ public static void setRippleDrawable(View view, int normalColor, int touchColor)
                 view.setBackground(stateListDrawable);
             }
         } catch (Exception e) {
-            Log.e("DEBUG", "" + e);
+
+
```
Comment (krharsh17) [inline library/src/main/java/in/krharsh17/barview/Bar.java]:
```suggestion
```
Please remove extra empty lines

File: library/src/main/java/in/krharsh17/barview/BarView.java [hunk 2]
```diff
@@ -101,14 +101,14 @@ public BarView(Context context, AttributeSet attrs) {
             final TypedArray a = context.obtainStyledAttributes(attrs,
                     R.styleable.BarView, 0, 0);
 
-            Log.i("DEBUG", "BarGroup: " + a.getInt(R.styleable.BarView_valueTextSize, 2));
+
```
Comment (krharsh17) [inline library/src/main/java/in/krharsh17/barview/BarView.java]: Please remove unnecessary empty line

File: library/src/main/java/in/krharsh17/barview/BarView.java [hunk 3]
```diff
@@ -101,14 +101,14 @@ public BarView(Context context, AttributeSet attrs) {
...
 
             VERTICAL_SPACING = a.getInteger(R.styleable.BarView_barGroupSpacing, VERTICAL_SPACING);
             BAR_HEIGHT = a.getInteger(R.styleable.BarView_barHeight, BAR_HEIGHT);
             LABEL_FONT_SIZE = a.getInteger(R.styleable.BarView_labelTextSize, LABEL_FONT_SIZE);
             VALUE_FONT_SIZE = a.getInteger(R.styleable.BarView_valueTextSize, VALUE_FONT_SIZE);
 
-            Log.i("DEBUG", "BarGroup: " + LABEL_FONT_SIZE);
+
```
Comment (krharsh17) [inline library/src/main/java/in/krharsh17/barview/BarView.java]: Please remove unnecessary empty line

**Review by krharsh17 — CHANGES_REQUESTED** [review]

**Comment by krharsh17** [comment]
Also, please mention the issue that you've submitted a PR for, as in this case #8 This will help in automatically closing the issue when the PR is merged!

**Comment by Medhavi-16** [comment]
Okay sir

**Review by krharsh17 — APPROVED** [review]

**Pull request merged**